### DISCUSSION TITLE
A website with interview experiences and questions asked in an interview.

### DISCUSSION THREAD

**Discussion by rithwik003** [scaleracademy/open-source-projects#125] [Ideas]
# Website with interview experiences and questions asked in an interview
- In this website the verified user can be able to add the experiences of an interview with the questions given in it.
- First the user need to get verified to post the experiences.
- By viewing this website many of the people got knowledge about the type of questions to be asked in an interview.
- By this website many of the job seekers will get benifited.
- It would be helpful for many if this is done.

**Comment by BhumikaJena** [comment]
Great idea !

**Comment by HarshithSimha** [comment]
Yes, it will be helpful for many job seekers. I appreciate your idea.
